django/db/models/fields/related.py

```diff
@@ -461,11 +461,16 @@ def _check_unique_target(self):
         except exceptions.FieldDoesNotExist:
             return []
 
+        nonempty_foreign_related_fields = tuple(rel_field
```

Comment:
I think it is kinda hiding symptoms instead of fixing actual problem.
If you will look at this method in django1.7 it is returning `[]` here https://github.com/django/django/blob/e03fe2f6bbad7305146d0e19e5e9a36eda2b982c/django/db/models/fields/related.py#L1344
If you will remove this return, it will fail with the same exception.
So, I think it should be fixed not there, but lets wait for some core dev who is more aware of this checks